generator/generator.yml

```diff
@@ -710,3 +710,16 @@ modules:
         type: EnumAsInfo
       "1.3.6.1.4.1.534.6.6.7.6.1.1.3":
         type: DisplayString
+
+# CyberPower
+#
+# https://www.cyberpowersystems.com/product/software/mib-files/mib-v29/
+  cyberpower:
+    version: 1
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
```

Comment:
Like I said, this needs to be removed.

```suggestion
```

Reply:
Could you elaborate on why?  These are things I want to receive in the stats data.  If these are now part of a default collection list, they're listed in almost every other device type in the file, should those be cleaned up?

I'd be nice to have context to understand requested changes.

Reply:
Hi @SuperQ, just wanted to follow up here and make sure I understand the request.

What is the intended way that a user could get both device MIB specific data (ie: PDU or UPS metrics) as well as network interface metrics on the same device?

I'm not a Go programmer, but it looks like the `module` parameter to the query can only take a single value as a string argument.  So would we need to create two scrape jobs in Prometheus?  One to get the UPS data and one to get the IF data?

Reply:
We're trying to clean up the duplication in the example configuration. You can get this data using the `if_mib` module as the default.

But this is just an example config, it's not meant to be every possible combination of MIBs / walks.

Reply:
> We're trying to clean up the duplication in the example configuration. You can get this data using the `if_mib` module as the default.
> 
> But this is just an example config, it's not meant to be every possible combination of MIBs / walks.

Thank you for the explanation, the PR has been updated.